### PR TITLE
Add Data Migration docs

### PR DESCRIPTION
**Pull request by nosan**
HZC-654

### CODE REVIEW

File: docs/modules/ROOT/pages/data-migration.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+When you connect publish cluster and try to write some data, you will see data will be replicated to consumer cluster.
+
+NOTE: You can synchronize all data by using `Sync` button in the WAN replication settings.
+
+== Connect to a Cluster
+
+After migration is done, you can connect to a new cluster via different clients, e.g. xref:java-client.adoc[]
+
+
+== Delete a Cluster
```
Comment (horakihor) [inline docs/modules/ROOT/pages/data-migration.adoc:51]: I think they need to delete WAN replication first?

**Review by horakihor — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/data-migration.adoc (hunk 1 above)
Comment (nosan) [reply docs/modules/ROOT/pages/data-migration.adoc:51]: No, they should completely migrate all clients first and only after a while delete wan and cluster itself.

Comment (horakihor) [reply docs/modules/ROOT/pages/data-migration.adoc:51]: Correct, but there is nothing about - "delete WAN". So I think it worth mentioning that WAN needs to be deleted and after that, you can delete the old cluster.

Comment (nosan) [reply docs/modules/ROOT/pages/data-migration.adoc:51]: done

**Review by horakihor — APPROVED** [review]

**Pull request merged**